scripts/statping.py

```diff
@@ -0,0 +1,27 @@
+import json
+import click
+
+
+@click.command()
+@click.option('--file-src', '-fs', 'file_path_src', required=True)
+@click.option('--token', '-t', 'token', required=True)
+@click.option('--file-dst', '-fd', 'file_path_dest', required=True)
```

Comment:
better add file type for option 

Check https://click.palletsprojects.com/en/8.1.x/arguments/#file-arguments